lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -206,6 +206,11 @@
   dnf:
     name: httpd
     state: latest
+    
+- name: install the latest version of Apache and MariaDB
+  dnf:
+    name: "httpd, mariadb-server"
+    state: latest
```

Comment:
Our preference is to use YAML lists, instead of comma separated values:

```
- name: install the latest version of Apache and MariaDB
   dnf:
     name:
       - httpd
       - mariadb-server
     state: latest
```

While we support CSV, YAML is recommended.

Reply:
I quickly tried, but not worked.
Retried correctly, works :+1: 

Commit updated